odysseus/halow_setup/install/nrc-firmware-flash

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
Why do we set the GPIO of the Pi for this? Can we actually run this since it is exe?

Reply:
Pin 13 is used for UART selection pin for Standalone mode and download mode, necessary for firmware flash (download mode) and AT command testing (standalone).  I got this directly from ALFA.  Exe is the source suffix, however it is normal javaFX (or swing not 100% sure as source is closed I believe).  I have not actually tested firmware flashing but the program launches well without errors.
I will add this description and a link to docs of the applications to the README and switch the gpio selection to something distro agnostic.